packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx

```diff
@@ -169,6 +179,19 @@ export const Default: StoryObj<CheckboxProps> = {
   args: {
     label: "Enable warp drive recalibration",
   },
+  render: function C({ label }) {
+    const [isChecked, setIsChecked] = useState(false);
+    return (
+      <Checkbox
+        label={label}
+        isChecked={isChecked}
+        onChange={(_, checked) => {
+          console.log("onChange", checked);
+          setIsChecked(checked);
```

Comment:
```suggestion
        onChange={(_, checked) => {
          console.log("onChange", isChecked);
          setIsChecked(isChecked);
```

Reply:
We don't need to `console.log` here. Storybook picks that up itself when `onChange` is called. If not, we need to use `action("onChange")(...args)` from Storybook instead of console logging.

Reply:
I'm not sure why it was originally added. I'll remove it.